acai-ios/BowlOrderDetailViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  BowlOrderDetailViewController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+protocol DidSelectOption: class {
+    func deselectOption(at index: Int)
+    func selectOption(at index: Int, type: OrderCustomizationOptionType)
+}
+
+class BowlOrderDetailViewController: UIViewController {
+    
+    // MARK: View vars
+    var collectionView: UICollectionView!
+    var listAdapter: ListAdapter!
+    
+    // MARK: Data
+    var baseOptions: [OrderCustomizationOption]!
+    var bowlItem: BowlItem!
+    var toppingOptions: [OrderCustomizationOption]!
+    
+    // MARK: View Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        // hard code
+        bowlItem = Acai.testBowl
+        toppingOptions = bowlItem.toppingOptions
+        baseOptions = bowlItem.baseOptions
+
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.backgroundColor = .white
+        collectionView.showsVerticalScrollIndicator = false
+        view.addSubview(collectionView)
+        
+        collectionView.snp.makeConstraints { (make) in
+            make.edges.equalToSuperview()
+        }
+        
+        listAdapter = ListAdapter(updater: ListAdapterUpdater(), viewController: self, workingRangeSize: 0)
+        listAdapter.collectionView = collectionView
+        listAdapter.dataSource = self
+    }
+    
+}
+
+extension BowlOrderDetailViewController: ListAdapterDataSource {
+    func objects(for listAdapter: ListAdapter) -> [ListDiffable] {
+        var list: [ListDiffable] = []
+        list.append(HeaderItem(title: "Choose your base") as ListDiffable)
+        list.append(OrderCustomizationOptions(options: baseOptions))
+        list.append(HeaderItem(title: "Choose your toppings") as ListDiffable)
+        list.append(OrderCustomizationOptions(options: toppingOptions))
+        return list
```

Comment:
you don't need to cast `as ListDiffable` since they're all `ListDiffable` models now.
also you can do this in one line
```
return [
  HeaderItem(title: "Choose your base"),
  ...
]
```